### PR TITLE
parameterise backup frequency to be user input

### PR DESCRIPTION
**Pull request by vijayabharathib**
fix issue #20

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,10 +22,19 @@ RUN chmod +x /usr/local/bin/dockerize
 ARG GIT_COMMIT_ID=unspecified
 ENV GIT_COMMIT_ID=$GIT_COMMIT_ID
 
+# Target frequencies needed. Make them true from docker compose. 
+
+ENV SCHEDULE_BACKUP_HOURLY "false"
```
Comment (arulrajnet) [inline Dockerfile]: @vijayabharathib can we set this as true by default?

File: .env [hunk 2]
```diff
@@ -11,3 +11,8 @@ MYSQL_PORT="3306"
 MYSQL_USER="root"
 MYSQL_PASSWORD="secret"
 MYSQLDUMP_DATABASE="my_wiki"
+# specify the frequencies needed
```
Comment (arulrajnet) [inline .env:14]: @vijayabharathib Here this has to be like `key=value` format. 

```
SCHEDULE_BACKUP_HOURLY="true"
```
like the above

**Review by arulrajnet — CHANGES_REQUESTED** [review]
Address the review comments.

File: Dockerfile (hunk 1 above)
Comment (vijayabharathib) [reply Dockerfile]: done.

File: .env (hunk 2 above)
Comment (vijayabharathib) [reply .env:14]: done.

**Review by arulrajnet — APPROVED** [review]

**Pull request merged**